dashdotdb/models/dashdotdb.py

```diff
@@ -121,8 +119,9 @@ class Vulnerability(db.Model):
     __tablename__ = 'vulnerability'
 
     id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String(64), unique=False, index=True)
+    name = db.Column(db.String(10000), unique=False, index=True)
     description = db.Column(db.String(10000), unique=False)
+    namespacename = db.Column(db.String(10000), unique=False)
```

Comment:
Is this namespace name in the sense of kubernetes? Shouldn't it have 63 characters max (or 253, I never remember 😂 )

Reply:
My understanding of namespacename is that it is more of a repository identifier for the package, like

```
RHEL8-rhel-8-including-unpatched
```

With 64 I ran into too small errors with the updated example [imagemanifestvuln.json](examples/imagemanifestvuln.json). 

However, it might as well be overkill. Will reduce to 512. If we run into issues we could still increase.

Reply:
Ok actually reduced it to 128 now.